web-app/src/app/tabs/controls/controls.component.css

```diff
@@ -0,0 +1,161 @@
+.header {
+	display: flex;
+	justify-content: space-between;
+	font-family: Arial, sans-serif;
+	font-size: 1.5vw;
+	background-color: transparent;
+	color: #D5FEFF;
+	padding-bottom: 0.2em;
+	padding-top: .2em;
+	margin-top: 0.1em;
+	margin-bottom: 0.1em;
+	width: auto;
+}
+.title-left {
+	float: left;
+	width: 50%;
+	margin-left: 5%;
+}
+.title-right {
+	color: #73a2ef;
+	float: left;
+	width: 5%;
+	margin-left: 5px;
+}
+.aliveGroup {
+	margin-left: auto;
+	width: auto;
+}
+.aliveGroup > .aliveText {
+	margin-left: 0.2em;
+}
+.circle {
+	float: left;
+	width: 1em;
+	height: 1em;
+}
+.circleAlive {
+	float: left;
+	width: 1em;
+	height: 1em;
+	margin-top: 2px;
+}
+.width {
+	width: 100%;
```

Comment:
I don't think this actually does anything

Reply:
If it isn't there, the whole page squishes to the left